openedx/tests/xblock_integration/test_external_xblocks.py

```diff
@@ -0,0 +1,37 @@
+"""
+This will run tests on all XBlocks in the `xblock.test.v0`
+entrypoint. Did you notice something about that entry point? It ends
+with a v0. That means this is not finished. At some point, we might
+stop running v0 tests, replacing them with test case failures, and
+run v1 tests only.
+
+That be the dragon here.
+"""
+import pkg_resources
+
+class DuplicateXBlockTest(Exception):
+    pass
+
+class InvalidTestName(Exception):
+    pass
+
+loaded = False
+
+for entrypoint in pkg_resources.iter_entry_points(group="xblock.test.v0"):
+    plugin = entrypoint.load()
+    classname = str(entrypoint).split(":")[-1]
```

Comment:
At this point, `plugin` is the class itself.  No need to parse and check for validity.  Use `classname = p.__name__`.